src/NzbDrone.Core/Notifications/Stash/StashProxy.cs

```diff
@@ -37,6 +51,7 @@ public void Update(StashSettings settings, string path)
                                 scanGeneratePhashes: {(settings.GeneratePhashes ? "true" : "false")},
                                 paths: [{cleanPath}]
                             }})
+                            {(settings.MetadataIdentify ? MetadataIdentifyQuery : "" )}
```

Comment:
This is the same logic as you already have above, no use in another conditional here